includes/razorpay-webhook.php

```diff
@@ -91,7 +110,14 @@ protected function paymentAuthorized($data)
 
         $razorpayPaymentId = $data['payload']['payment']['entity']['id'];
 
-        $payment = $this->api->payment->fetch($razorpayPaymentId);
+        try
+        {
+            $payment = $this->api->payment->fetch($razorpayPaymentId);
+        }
+        catch (Exception $e)
+        {
+            echo "Razorpay Error: " . $e->getMessage();
```

Comment:
echoing in a webhook achieves nothing. Either throw exceptions and catch it somewhere in/above the process function, or log+exit on errors